src/qt/bitcoingui.cpp

```diff
@@ -572,6 +573,12 @@ bool BitcoinGUI::setCurrentWallet(const QString& name)
     return walletFrame->setCurrentWallet(name);
 }
 
+bool BitcoinGUI::setCurrentWalletBySelectorIndex(int index)
+{
+    QString internal_name = m_wallet_selector->itemData(index).toString();
```

Comment:
nit, explicitly select `Qt::UserRole`?

Reply:
I don't think this is necessary,... or what would be the benefits of explicitly set it? I don't expect multiple roles in that dropdown.